components/automate-gateway/handler/authz_test.go

```diff
@@ -181,11 +181,11 @@ func TestIntrospectAllProjectsV2(t *testing.T) {
 	}{
 		"empty response": {
 			&authz_v2.FilterAuthorizedProjectsResp{},
-			[]string{},
+			[]string(nil),
```

Comment:
ℹ️ By writing `[]string(nil)`, you're "typing `nil`" -- and we don't need that, its type is clear from [the struct definition](https://github.com/chef/automate/pull/416/files/4d8ce51c08272cc3d1b0202a81ccf420cf4773c2..7c509bccf2eddae87ba6ef588037a58c42b47b92#diff-e23e2853bf09746e5ad1182dbabfa42bR180).

[See this playground for some illustration.](https://play.golang.org/p/0ZGC0DiTSN5)

Reply:
This change was introduced to make the test pass. Are you suggesting there is a better way?

Reply:
Yeah just try `nil` there now

Reply:
Changed it there, and in a few other places as well.